src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionStringBuilder.cs

```diff
@@ -169,1766 +176,1672 @@ static SqlConnectionStringBuilder()
+                { DbConnectionStringSynonyms.CONNECTIONTIMEOUT, Keywords.ConnectTimeout },
+                { DbConnectionStringSynonyms.CONNECTRETRYCOUNT, Keywords.ConnectRetryCount },
+                { DbConnectionStringSynonyms.CONNECTRETRYINTERVAL, Keywords.ConnectRetryInterval },
+                { DbConnectionStringSynonyms.TIMEOUT, Keywords.ConnectTimeout },
+                { DbConnectionStringSynonyms.LANGUAGE, Keywords.CurrentLanguage },
+                { DbConnectionStringSynonyms.ADDR, Keywords.DataSource },
+                { DbConnectionStringSynonyms.ADDRESS, Keywords.DataSource },
+                { DbConnectionStringSynonyms.MULTIPLEACTIVERESULTSETS, Keywords.MultipleActiveResultSets },
+                { DbConnectionStringSynonyms.MULTISUBNETFAILOVER, Keywords.MultiSubnetFailover },
+                { DbConnectionStringSynonyms.NETWORKADDRESS, Keywords.DataSource },
+                { DbConnectionStringSynonyms.POOLBLOCKINGPERIOD, Keywords.PoolBlockingPeriod },
+                { DbConnectionStringSynonyms.SERVER, Keywords.DataSource },
+                { DbConnectionStringSynonyms.DATABASE, Keywords.InitialCatalog },
+                { DbConnectionStringSynonyms.TRUSTEDCONNECTION, Keywords.IntegratedSecurity },
+                { DbConnectionStringSynonyms.TRUSTSERVERCERTIFICATE, Keywords.TrustServerCertificate },
+                { DbConnectionStringSynonyms.ConnectionLifetime, Keywords.LoadBalanceTimeout },
+                { DbConnectionStringSynonyms.Pwd, Keywords.Password },
+                { DbConnectionStringSynonyms.PERSISTSECURITYINFO, Keywords.PersistSecurityInfo },
+                { DbConnectionStringSynonyms.UID, Keywords.UserID },
+                { DbConnectionStringSynonyms.User, Keywords.UserID },
+                { DbConnectionStringSynonyms.WSID, Keywords.WorkstationID }
+            };
+            Debug.Assert((KeywordsCount + SqlConnectionString.SynonymCount) == pairs.Count, "initial expected size is incorrect");
+            return pairs;
+        }
+
+        private static bool ConvertToBoolean(object value) => DbConnectionStringBuilderUtil.ConvertToBoolean(value);
+
+        private static int ConvertToInt32(object value) => DbConnectionStringBuilderUtil.ConvertToInt32(value);
+
+        private static bool ConvertToIntegratedSecurity(object value) => DbConnectionStringBuilderUtil.ConvertToIntegratedSecurity(value);
+
+        private static SqlAuthenticationMethod ConvertToAuthenticationType(string keyword, object value) => DbConnectionStringBuilderUtil.ConvertToAuthenticationType(keyword, value);
+
+        private static string ConvertToString(object value) => DbConnectionStringBuilderUtil.ConvertToString(value);
+
+        private static ApplicationIntent ConvertToApplicationIntent(string keyword, object value) => DbConnectionStringBuilderUtil.ConvertToApplicationIntent(keyword, value);
+
+        /// <summary>
```

Comment:
These xml comments for the private method aren't really helpful, let's remove them?